### PR TITLE
Test storageManager

### PR DESCRIPTION
**Pull request by fynsta**
Also adding the case for double and typing `readData` better

### CODE REVIEW

File: test/models/chat_test.dart [hunk 1]
```diff
@@ -33,7 +33,11 @@ void main() {
       senderId: profileId + 1,
       createDependencies: false,
     );
-    SupabaseManager.setCurrentProfile(profile);
+
+    setUp(() {
+      SupabaseManager.setCurrentProfile(profile);
```
Comment (fynsta) [inline test/models/chat_test.dart:38]: @istzen another error where `setUp` was missing, although I'm not sure why this did not happen before?

**Comment by Eddie-42** [comment]
> # [Codecov](https://codecov.io/gh/MOTIS-Mitfahr-App/flutter-app/pull/108?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=MOTIS-Mitfahr-App) Report
> > Merging [#108](https://codecov.io/gh/MOTIS-Mitfahr-App/flutter-app/pull/108?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=MOTIS-Mitfahr-App) ([e04f513](https://github.com/MOTIS-Mitfahr-App/flutter-app/commit/e04f51356ad99d000b133c096c13e1db2938ef3e)) into [main](https://codecov.io/gh/MOTIS-Mitfahr-App/flutter-app/commit/0ee0c625b9300608d565b9fd278d10bf70a812ce?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=MOTIS-Mitfahr-App) ([0ee0c62](https://github.com/MOTIS-Mitfahr-App/flutter-app/commit/0ee0c625b9300608d565b9fd278d10bf70a812ce)) will **increase** coverage by `0.19%`.
> > The diff coverage is `71.42%`.
> 
> Additional details and impacted files

What a banger :D

File: lib/util/storage_manager.dart [hunk 2]
```diff
@@ -5,6 +5,8 @@ class StorageManager {
     final SharedPreferences prefs = await SharedPreferences.getInstance();
     if (value is int) {
       await prefs.setInt(key, value);
+    } else if (value is double) {
```
Comment (Eddie-42) [inline lib/util/storage_manager.dart:8]: Do we need that anywhere or did the exclusion just bug you?

**Review by Eddie-42 — APPROVED** [review]

File: lib/util/storage_manager.dart (hunk 2 above)
Comment (fynsta) [reply lib/util/storage_manager.dart:8]: Yes that was just annoying lol - we listed all the data types that `SharedPreferences` supports, __except__ double.

File: test/models/chat_test.dart (hunk 1 above)
Comment (istzen) [reply test/models/chat_test.dart:38]: thanks and sorry

**Review by istzen — APPROVED** [review]

**Review by Showcas — APPROVED** [review]

**Pull request merged**